Syntaxes/Liquid-HTML.sublime-syntax

```diff
@@ -28,6 +28,14 @@ contexts:
               scope: punctuation.definition.comment.end.liquid
               pop: true
 
+        - match: '{%-?\s*schema\s*-?%}'
+          scope: schema.liquid punctuation.section.schema.begin.liquid
+          embed: scope:source.json
+          embed_scope: schema.liquid
```

Comment:
Take a look here: https://github.com/sublimehq/Packages/blob/master/Markdown/Markdown.sublime-syntax#L1117

I think the `embed_scope` should perhaps be something like `markup.json.schema.liquid`?